### ISSUE TITLE
ImportError: No module named 'tornado'

### ISSUE THREAD

**Issue by jonrossmacmillan** [instana/python-sensor#151]
We started getting this error in our CI today, which is configured to get the latest version of the instana package. 
  
```  sys.exit(run())
  File "/opt/miniconda/envs/.venv/lib/python3.5/site-packages/gunicorn/app/wsgiapp.py", line 61, in run
    WSGIApplication("%(prog)s [OPTIONS] [APP_MODULE]").run()
  File "/opt/miniconda/envs/.venv/lib/python3.5/site-packages/gunicorn/app/base.py", line 223, in run
    super(Application, self).run()
  File "/opt/miniconda/envs/.venv/lib/python3.5/site-packages/gunicorn/app/base.py", line 72, in run
    Arbiter(self).run()
  File "/opt/miniconda/envs/.venv/lib/python3.5/site-packages/gunicorn/arbiter.py", line 60, in __init__
    self.setup(app)
  File "/opt/miniconda/envs/.venv/lib/python3.5/site-packages/gunicorn/arbiter.py", line 95, in setup
    self.log = self.cfg.logger_class(app.cfg)
  File "/opt/miniconda/envs/.venv/lib/python3.5/site-packages/gunicorn/glogging.py", line 200, in __init__
    self.setup(cfg)
  File "/opt/miniconda/envs/.venv/lib/python3.5/site-packages/gunicorn/glogging.py", line 256, in setup
    disable_existing_loggers=False)
  File "/opt/miniconda/envs/.venv/lib/python3.5/logging/config.py", line 76, in fileConfig
    formatters = _create_formatters(cp)
  File "/opt/miniconda/envs/.venv/lib/python3.5/logging/config.py", line 123, in _create_formatters
    c = _resolve(class_name)
  File "/opt/miniconda/envs/.venv/lib/python3.5/logging/config.py", line 94, in _resolve
    found = __import__(used)
  File "/usr/local/pythagoras/pythagoras/__init__.py", line 1, in <module>
    from flask import Flask
  File "/opt/miniconda/envs/.venv/lib/python3.5/site-packages/wrapt/importer.py", line 158, in load_module
    notify_module_loaded(module)
  File "/opt/miniconda/envs/.venv/lib/python3.5/site-packages/wrapt/decorators.py", line 440, in _synchronized
    return wrapped(*args, **kwargs)
  File "/opt/miniconda/envs/.venv/lib/python3.5/site-packages/wrapt/importer.py", line 136, in notify_module_loaded
    hook(module)
  File "/opt/miniconda/envs/.venv/lib/python3.5/site-packages/wrapt/importer.py", line 105, in import_hook
    __import__(entrypoint.module_name)
  File "/opt/miniconda/envs/.venv/lib/python3.5/site-packages/instana/__init__.py", line 90, in <module>
    boot_agent()
  File "/opt/miniconda/envs/.venv/lib/python3.5/site-packages/instana/__init__.py", line 65, in boot_agent
    import instana.singletons
  File "/opt/miniconda/envs/.venv/lib/python3.5/site-packages/instana/singletons.py", line 28, in <module>
    from opentracing.scope_managers.tornado import TornadoScopeManager
  File "/opt/miniconda/envs/.venv/lib/python3.5/site-packages/opentracing/scope_managers/tornado.py", line 24, in <module>
    import tornado.stack_context
ImportError: No module named 'tornado'

```

our conda-requirements.yaml file:

```
name: .venv
channels:
  - defaults
  - conda-forge
dependencies:
  - asn1crypto=0.24.0=py35_3
  - ca-certificates=2018.11.29=ha4d7672_0
  - certifi=2018.8.24=py35_1001
  - cffi=1.11.5=py35h5e8e0c9_1
  - chardet=3.0.4=py35_3
  - click=7.0=py_0
  - cryptography=2.3.1=py35hdffb7b8_0
  - cryptography-vectors=2.3.1=py35_0
  - fastcache=1.0.2=py35h470a237_1
  - flask=1.0.2=py_2
  - gevent=1.3.6=py35h470a237_0
  - gmp=6.1.2=hf484d3e_1000
  - gmpy2=2.0.8=py35_0
  - gunicorn=19.9.0=py35_1000
  - idna=2.7=py35_2
  - itsdangerous=1.1.0=py_0
  - jinja2=2.10=py_1
  # - libcxx=7.0.0=h6bb024c_1002
  - libedit
  - libffi=3.2.1=hf484d3e_1005
  - libgcc-ng=7.3.0=hdf63c60_0
  - libstdcxx-ng=7.3.0=hdf63c60_0
  - llvm-meta=7.0.0=0
  - markupsafe=1.0=py35h470a237_1
  - mpc=1.1.0=hb20f59a_1006
  - mpfr=4.0.1=ha14ba45_1000
  - mpmath=1.1.0=py_0
  - ncurses=6.1=hf484d3e_1002
  - openssl=1.0.2p=h14c3975_1002
  - pkg-config=0.29.2=h1bed415_8
  - pip=18.0=py35_1001
  - pycparser=2.19=py_0
  - pyopenssl=18.0.0=py35_0
  - pysocks=1.6.8=py35_2
  - pyyaml=3.13=py35h470a237_1
  - readline=7.0=hf8c457e_1001
  - requests=2.19.1=py35_1
  - setuptools=40.4.3=py35_0
  - six=1.11.0=py35_1
  - sqlite=3.26.0=h67949de_1000
  - structlog=19.1.0=py_0
  - sympy=1.1.1=py35_0
  - tk=8.6.9=h84994c4_1000
  - urllib3=1.23=py35_1
  - werkzeug=0.14.1=py_0
  - wheel=0.32.0=py35_1000
  - xz=5.2.4=h14c3975_1001
  - yaml=0.1.7=h14c3975_1001
  - zlib=1.2.11=h14c3975_1004
  - greenlet=0.4.14=py35h14c3975_0
  - python=3.5.6=hc3d631a_0
  - pip:
    - autowrapt==1.0
    - basictracer==3.0.0
    - fysom==2.1.5
    - instana
    - json-logging-py==0.2
    - opentracing==2.0.0
    - protobuf==3.7.0rc2
    - wrapt==1.11.1
prefix: /opt/miniconda/envs/.venv
```

**Comment by pglombardo** [comment]
@jonrossmacmillan  - thanks for reporting this.  We released package 1.11.1 that should resolve this.  Could you confirm once you get a chance?

https://github.com/instana/python-sensor/releases/tag/1.11.1

**Comment by jonrossmacmillan** [comment]
That change fix it. Thanks!
